dockers/docker-sonic-mgmt/Dockerfile.j2

```diff
@@ -5,6 +5,7 @@ ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update && apt-get install -y apt-transport-https \
                                          apt-utils \
+                                         azure-cli \
```

Comment:
@shbalaku-microsoft Version of the azure-cli package installed from apt is much lower than the package installed by one command. 
`2.0.81` vs `2.49.0`

It is better to keep the current way of installing `azure-cli`.

Reply:
I see, this is to address the build issue https://github.com/sonic-net/sonic-buildimage/issues/15567